src/Microsoft.Framework.Runtime/NuGet/Utility/VersionUtility.cs

```diff
@@ -80,6 +88,9 @@ public static class VersionUtility
         // These aliases allow us to accept 'wp', 'wp70', 'wp71', 'windows', 'windows8' as valid target farmework folders.
         private static readonly Dictionary<FrameworkName, FrameworkName> _frameworkNameAlias = new Dictionary<FrameworkName, FrameworkName>(FrameworkNameEqualityComparer.Default)
         {
+            { new FrameworkName("Asp.Net, Version=v5.0"), new FrameworkName("DNX, Version=v4.5.1") },
```

Comment:
If you kpm build either of aspnetcore50 or dnxcore50, it will only build the section with that exact name. We can clean that up a bit. See #1344